module/applications/actor/config/ability-config.mjs

```diff
@@ -0,0 +1,66 @@
+import BaseConfigSheet from "../api/base-config-sheet.mjs";
+
+/**
+ * Configuration application for an actor's abilities.
+ */
+export default class AbilityConfig extends BaseConfigSheet {
+  /** @override */
+  static DEFAULT_OPTIONS = {
+    ability: null,
+    position: {
+      width: 500
+    }
+  };
+
+  /** @override */
+  static PARTS = {
+    config: {
+      template: "systems/dnd5e/templates/actors/config/ability-config.hbs"
+    }
+  };
+
+  /* -------------------------------------------- */
+  /*  Properties                                  */
+  /* -------------------------------------------- */
+
+  /**
+   * Configuration data for the ability being edited.
+   * @type {AbilityConfiguration}
+   */
+  get abilityConfig() {
+    return CONFIG.DND5E.abilities[this.options.ability];
+  }
+
+  /* -------------------------------------------- */
+
+  /** @override */
+  get title() {
+    return game.i18n.format("DND5E.ABILITY.Configure.Title", { ability: this.abilityConfig?.label });
+  }
+
+  /* -------------------------------------------- */
+  /*  Rendering                                   */
+  /* -------------------------------------------- */
+
+  /** @inheritDoc */
+  async _preparePartContext(partId, context, options) {
+    context = await super._preparePartContext(partId, context, options);
+    const source = this.document.system.toObject();
```

Comment:
Something I've been trying to be more aware of, is that we can pass `this.document.system._source` directly (assuming we never mutate it, which I think is the case here), rather than calling `toObject`, which should make our rendering and re-rendering a bit faster.